src/containers/Surah/index.js

```diff
@@ -447,7 +447,7 @@ class Surah extends Component {
           <Audioplayer
             chapter={chapter}
             verses={verses}
-            currentVerse={verses[currentVerse]}
+            currentVerse={verses[currentVerse] || verses[Object.keys(verses)[0]]}
```

Comment:
I guess we're doing the same thing in `mapStateToProps`, currentVerse shouldn't be nil. 

https://github.com/imadmk/quran.com-frontend/blob/f178b4c43fae01bd3f549d8c12aaa0c167943346/src/containers/Surah/index.js#L497

Also this will fix the audio player what about other components which are using `currentVerse` ? see `renderVerses ` and `renderLines` for example. I believe we should fix `mapStateToProps` and make sure current verse isn't undefine at all.